src/collective/nitf/upgrades/v2000/__init__.py

```diff
@@ -64,3 +66,26 @@ def update_configlet(setup_tool):
     profile = 'profile-{0}:default'.format(PROJECTNAME)
     setup_tool.runImportStepFromProfile(profile, 'controlpanel')
     logger.info('Control panel configlet updated.')
+
+
+def update_behavior(setup_tool):
+    """Update behavior."""
+    fti = queryUtility(IDexterityFTI, name='collective.nitf.content')
+    behaviors = list(fti.behaviors)
+    behaviors.remove('plone.app.referenceablebehavior.referenceable.IReferenceable')
```

Comment:
you have to check if the behaviors are applied or not before adding or removing them,